src/app/activity/bounty/Create.ts

```diff
@@ -130,12 +132,23 @@ export const createBounty = async (createRequest: CreateRequest): Promise<any> =
             const dmChannel: DMChannel = await walletNeededMessage.channel.fetch() as DMChannel;
             
             await createRequest.commandContext.send({content: `Waiting for <@${owedTo.id}> to enter their wallet address.`, ephemeral: true});
-            if (! (await BountyUtils.userInputWalletAddress(dmChannel, createRequest.userId, durationMinutes*60*1000))) {
-                owedTo.send(
-                `Unable to complete this operation due to timeout or incorrect wallet addresses.\n` +
-                'Please try entering your wallet address with the slash command `/register wallet`.\n\n' +
-                `Return to Bounty list: ${(await BountyUtils.getLatestCustomerList(createRequest.guildId))}`
-                );
+
+            try {
+                await BountyUtils.userInputWalletAddress(dmChannel, createRequest.userId, durationMinutes*60*1000);
+                await createRequest.commandContext.delete();
+            }
+            catch (e) {
+                if (e instanceof TimeoutError) {
```

Comment:
If the user fails after 3 tries, it just quits with no other message. I think you need to catch the validation error here

Reply:
Absolutely yes - fat fingered and left out the second part of that condition which checks for a ValidationError. Fixed on next revision, thanks!